lib/web_ui/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ environment:
   sdk: '>=3.6.0-0 <4.0.0'
 
 dependencies:
-  js: 0.6.4
+  js: ^0.7.0
```

Comment:
I wonder if deprecations will start showing up then?

Reply:
What deprecations are you referring to?

Reply:
Oh snap. I swore we marked libraries in pkg:js deprecated. Maybe not...